tests/core/io/test_marshalls.h

```diff
@@ -390,11 +390,30 @@ TEST_CASE("[Marshalls] Typed array decoding") {
 	Variant variant;
 	int r_len;
 	uint8_t buffer[] = {
-		0x1c, 0x00, 0x01, 0x00, // Variant::ARRAY, CONTAINER_TYPE_KIND_BUILTIN
-		0x02, 0x00, 0x00, 0x00, // Array type (Variant::INT).
-		0x01, 0x00, 0x00, 0x00, // Array size.
-		0x02, 0x00, 0x01, 0x00, // Element type (Variant::INT, HEADER_DATA_FLAG_64).
-		0xef, 0xcd, 0xab, 0x89, 0x67, 0x45, 0x23, 0xf1, // Element value.
+		0x1c,
+		0x00,
+		0x01,
+		0x00, // Variant::ARRAY, CONTAINER_TYPE_KIND_BUILTIN
+		0x02,
+		0x00,
+		0x00,
+		0x00, // Array type (Variant::INT).
+		0x01,
+		0x00,
+		0x00,
+		0x00, // Array size.
+		0x02,
+		0x00,
+		0x01,
+		0x00, // Element type (Variant::INT, HEADER_DATA_FLAG_64).
+		0xef,
+		0xcd,
+		0xab,
+		0x89,
+		0x67,
+		0x45,
+		0x23,
+		0xf1, // Element value.
```

Comment:
Use `// clang-format off` and `// clang-format on` around those to preserve the original formatting.